ui/admin/app/components/form/credential/ssh_private_key/index.hbs

```diff
@@ -66,24 +66,17 @@
             @layout='wide'
           />
         </form.radioGroup>
-      {{else}}
-        <form.radioGroup
-          @name='types'
-          @selectedValue={{@model.type}}
-          as |radioGroup|
-        >
-          <radioGroup.card
-            @label={{t (concat 'resources.credential.types.' @model.type)}}
-            @helperText={{t (concat 'resources.credential.help.' @model.type)}}
-            @value={{@model.type}}
-            @layout='wide'
-            @disabled={{true}}
-            readonly={{true}}
-          />
-        </form.radioGroup>
-      {{/if}}
-    </:body>
-  </form.fieldset>
+      </:body>
+    </form.fieldset>
+  {{else}}
+    <form.input
+      @name='type'
+      @label={{t 'form.type.label'}}
+      @value={{t (concat 'resources.credential.types.' @model.type)}}
+      @disabled={{true}}
```

Comment:
Is disabled necessary if it's readonly?

Reply:
@randallmorey not it is not.  thanks for the catch